certified-connectors/HTML To PDF By Pascalcase/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#18186d",
+    "capabilities": [],
+    "publisher": "Vikrant Upadhyay"

```

Comment:
@VikrantUpadhyay1 for all our Verified Publishers/API Owners we ask to have the "publisher" and the "stackOwner" set to the same company name. 
```
"publisher": "Name of company or organization.",
"stackOwner": "Name of the owner of the service or product."
```

Reply:
@Amjed-Ayoub thank you for your feedback, I have made all the changes which were asked by you.